### ISSUE TITLE
WorldWindow shutdown causes JVM crash on macOS

### ISSUE THREAD

**Issue by zglueck** [NASAWorldWind/WorldWindJava#54]
Running the Shutdown example and clicking the "Shutdown Window" will cause a JVM crash on macOS (contents of the crash message):

```
./run-demo.bash gov.nasa.worldwindx.examples.Shutdown
Running gov.nasa.worldwindx.examples.Shutdown
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fff8975b4dd, pid=741, tid=0x0000000000000a0b
#
# JRE version: Java(TM) SE Runtime Environment (8.0_102-b14) (build 1.8.0_102-b14)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.102-b14 mixed mode bsd-amd64 compressed oops)
# Problematic frame:
# C  [libobjc.A.dylib+0x14dd]  objc_msgSend+0x1d
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /hs_err_pid741.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
./run-demo.bash: line 16:   741 Abort trap: 6           java -Xmx512m -Dsun.java2d.noddraw=true -classpath ./src:./classes:./worldwind.jar:./worldwindx.jar:./jogl-all.jar:./gluegen-rt.jar:./gdal.jar $*
```

**Comment by zglueck** [comment]
Additionally, the issue can be observed in the TabbedPaneUsage demo when pressing "Attach" on the dummy pane (crash message contents):

Adding tab...#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fff8975b4dd, pid=749, tid=0x0000000000000a0b
#
# JRE version: Java(TM) SE Runtime Environment (8.0_112-b16) (build 1.8.0_112-b16)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.112-b16 mixed mode bsd-amd64 compressed oops)
# Problematic frame:
# C  [libobjc.A.dylib+0x14dd]  objc_msgSend+0x1d
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /hs_err_pid749.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

**Comment by zglueck** [comment]
Crash observed when using JRE 1.8.0_112 but not JRE 1.7.0_79

**Comment by zglueck** [comment]
Known work around is to use a JRE 7 on macOS. We will address this when #61 is resolved, the upgrade to JOGL v2.3.2

**Comment by markpet49** [comment]
WWJ has been upgraded to JOGL 2.4 which addresses this crash.
